react-bootstrap/src/index.js

```diff
@@ -17,33 +16,41 @@ const bs4test = () => {
 		el.setAttribute("class", "d-inline");
 
 		document.body.appendChild(el);
-		result = getComputedStyle(el).display === "inline";
+		result = window.getComputedStyle(el).display === "inline";
 		document.body.removeChild(el);
 	}
 
 	return result;
 };
 
 export const RunlyReactBootstrapProvider = ({ children, ...props }) => {
+	return (
+		<>
+			<BootstrapStyles />
+			<RunlyProvider {...props}>{children}</RunlyProvider>
+		</>
+	);
+};
+
+const BootstrapStyles = () => {
```

Comment:
After thinking about this a little more, I think we should export this component and not try to wrap it up with the Provider. We should just re-export the `RunlyProvider` like normal.

This is after me trying to get bootstrap to load separately from material in the `examples/gatsby` project. We need to be able to compose the components.

Reply:
How are we envisioning this working at the higher API layers?

Reply:
Ok, how about you export `BootstrapStyles`, and also export the `RunlyReactBootstrapProvider`, but just call it `RunlyProvider`. That way it will `BootstrapStyles` will be included by default when using `RunlyProvider` unless they explicitly import it from `core` in which case they can use `BootstrapStyles` separately.

Reply:
I was thinking the html/vanilla js APIs could take a `data-runly-theme`/`runlyTheme` which would determine which components to load.